sdcm/nemesis.py

```diff
@@ -3009,7 +3056,19 @@ def disrupt_network_random_interruptions(self):  # pylint: disable=invalid-name
             self.target_node.traffic_control(None)
             self._wait_all_nodes_un()
 
+    def _disrupt_network_block_k8s(self):
+        duration_options = ["10s", "60s", "120s", "300s", "500s"]
```

Comment:
Maybe it is better to get the `list_of_timeout_options` parameter with the values which are defined now on the line 3079?

In this case we will have just 1 place where it must be changed and it will reduce future possible mismatch for k8s and non-k8s backends.

WDYT?

Reply:
seems reasonable, fixed.